apache.yml

```diff
@@ -12,46 +12,27 @@ resource:
       tags:
         Name: '${var.koding_user_username}-${var.koding_group_slug}'
       user_data: |-
-        set -e
+        #!/bin/bash
+
+        set -xe
```

Comment:
i disagree with you. `set -euo pipefail` is a typical use.  therefore i think that there is no need to use the long option names. but, if Koding is using it in this way, there is no problem. I can fix it.